### PR TITLE
Add disable-{japi,vom,papi} flag

### PR DESCRIPTION
**Pull request by tiewei**
Disable compiling of java api, python api and vpp object model
bindings. This will save compile time.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ RUN . /build.env \
  && cd /vpp \
  && git pull \
  && git checkout ${VPP_COMMIT} \
- && UNATTENDED=y make install-dep bootstrap pkg-deb \
+ && UNATTENDED=y vpp_configure_args_vpp='--disable-japi --disable-vom --disable-papi' make install-dep bootstrap pkg-deb \
```
Comment (ondrej-fabry) [inline Dockerfile:20]: I think `vpp_configure_args_vpp='...'` part should be used as argument to `make`, as I showed in #3 and not before the make. 

I just tested it locally and it seems like it's ignoring it if it's not used as argument to make and makes ~10min time difference in compilation time.

Comment (unclejack) [reply Dockerfile:20]: This didn't make any difference for me. Can you provide the exact command you've used to run `make install-dep bootstrap pkg-deb`, please?

Comment (ondrej-fabry) [reply Dockerfile:20]: I ran it using: `make vpp_configure_args_vpp='--disable-japi --disable-vom --disable-papi' install-dep bootstrap pkg-deb`

**Comment by tiewei** [comment]
close this ticket for now, I saw some improvement on docker cloud, will include it in #7

**Pull request closed without merging**